app/main/controller/video_similarity_controller.py

```diff
@@ -17,7 +17,7 @@ class VideoSimilarityResource(Resource):
     @api.doc('Delete a video in the similarity database')
     @api.expect(video_similarity_request, validate=True)
     def delete(self):
-        return similarity.delete_item(request.args, "video")
+        return similarity.delete_item(request.json, "video") # Changed Since 4126 PR
```

Comment:
Same comment as above. Let's verify there are no DELETE requests with query-string arguments rather than bodies. I doubt there are, but worth checking.